lib/src/main/kotlin/io/provenance/plugins/keystone/model/SignatureRequest.kt

```diff
@@ -0,0 +1,12 @@
+package io.provenance.plugins.keystone.model
+
+data class SignatureRequest(
+    val data: ByteArray,
+    val signingType: SigningType,
+)
+
+enum class SigningType {
```

Comment:
what's the difference between these two signing types in Keystone?

Reply:
note entirely sure, but looks like p8e is unimplemented https://github.com/FigureTechnologies/provenance-keystone/blob/ee610b7fec1ed51ad2472ed568e8ebd5e89c6063/service/src/main/kotlin/io/provenance/keystone/service/KeystoneService.kt#L177